### PR TITLE
Lesson05

### PR DESCRIPTION
**Pull request by Vl92**

### CODE REVIEW

File: Lesson05/HW05.IntuitionTraining/Program.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            while(true)
+            {
+                Random random = new Random();
+                int returnValue = random.Next(1, 5);
+                int Guess = 0;
+                Console.WriteLine("Я загадал число от 1 до 5. Сможешь угадать?");
+
+
+
+                Guess = Convert.ToInt32(Console.ReadLine());
```
Comment (Onix-one) [inline Lesson05/HW05.IntuitionTraining/Program.cs]: Why are there so many spaces?

File: Lesson05/HW05.IntuitionTraining/Program.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    {       
+        public static object Guess { get; private set; }
+
+        static void Main(string[] args)
+        {
+            while(true)
+            {
+                Random random = new Random();
+                int returnValue = random.Next(1, 5);
+                int Guess = 0;
```
Comment (Onix-one) [inline Lesson05/HW05.IntuitionTraining/Program.cs]: The second time I say. Read about variable naming at Microsoft documentation...

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: Lesson05/HW05.SchoolControls/Program.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+namespace HW05.SchoolControls
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            while (true)
+            {
+                Console.WriteLine("Введите номер контрольного теста");
```
Comment (Onix-one) [inline Lesson05/HW05.SchoolControls/Program.cs:11]: It is desirable to write the internal text in English.

File: Lesson05/HW05.SchoolControls/Program.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                int numtest = 0;
+                numtest = Convert.ToInt32(Console.ReadLine());
+
+
+                if ((numtest >= 1) && (numtest <= 15))
```
Comment (Onix-one) [inline Lesson05/HW05.SchoolControls/Program.cs]: Give readable variable names and read the rules for naming variables again.

**Review by Onix-one — CHANGES_REQUESTED** [review]

File: Lesson05/HW05.IntuitionTraining/Program.cs (hunk 1 above)
Comment (Vl92) [reply Lesson05/HW05.IntuitionTraining/Program.cs]: Done

File: Lesson05/HW05.SchoolControls/Program.cs (hunk 4 above)
Comment (Vl92) [reply Lesson05/HW05.SchoolControls/Program.cs]: done

File: Lesson05/HW05.SchoolControls/Program.cs (hunk 3 above)
Comment (Vl92) [reply Lesson05/HW05.SchoolControls/Program.cs:11]: done

File: Lesson05/HW05.IntuitionTraining/Program.cs (hunk 2 above)
Comment (Vl92) [reply Lesson05/HW05.IntuitionTraining/Program.cs]: done

**Pull request merged**